runner/status.go

```diff
@@ -26,12 +26,12 @@ const (
 	ABORTED
 	// Operation timed out and was killed.
 	TIMEDOUT
-	// Invalid or error'd request. Original runner state not affected. Retry may work after mutation.
+	// Request rejected for reasons unrelated to the user.
```

Comment:
I'm not sure we have clear semantics on FAILED vs BADREQUEST...
if BADREQUEST is unrelated to user (task definition), then is FAILED related to user (task definition)?
FAILED = invalid command (os execer hits an error from Start(), invalid snapshot (checkout cannot find snapshot id)
BADREQUEST = the runner code hits an error that is other than listed for FAILED.....
The current code appears to be reporting FAILED from invoke.go for:
  . checkout returns error
  . creating stdout directory
  . creating stderr directory
  . execer.exec() returns an error
  . create tmp invoke dir
  . ingesting stdout/stderr
  . unexpected state from execer
(this doesn't align with the definition above)

We need to clarify the definitions (update the comments) and verify that the code matches.